katello/katello/hostname-change.rb

```diff
@@ -256,8 +256,12 @@ def run
 
       public_dir = "/var/www/html/pub"
       public_backup_dir = "#{public_dir}/#{@old_hostname}-#{self.timestamp}.backup"
-      STDOUT.puts "deleting old certs"
 
+      STDOUT.puts "removing old cert rpms"
+      cert_packages = ["#{@old_hostname}-foreman-proxy-client*", "#{@old_hostname}-qpid-client-cert*", "#{@old_hostname}-qpid-router-server*", "#{@old_hostname}-tomcat*", "#{@old_hostname}-apache*", "#{@old_hostname}-qpid-broker*", "#{@old_hostname}-puppet-client*", "#{@old_hostname}-foreman-proxy*", "#{@old_hostname}-qpid-router-client*", "#{@old_hostname}-foreman-client*"]
```

Comment:
Even though this file isn't the best about long line lengths, can you break up the length of this?